chef/cookbooks/crowbar/templates/default/crowbar-evacuate-host.sh.erb

```diff
@@ -47,7 +47,7 @@ fi
 
 # FIXME: here we need checks for potential errors during the evacuation
 
-while nova list --host "$host" --fields host 2>/dev/null | grep "$host"; do
+while nova list --all-tenants --status active --host "$host" --fields host 2>/dev/null | grep "$host"; do
```

Comment:
Not sure about status active. What about instances that are shutting off? They are still running until they finally shut off so they should be taken into account?